src/app_service/service/message/dto/message.nim

```diff
@@ -235,6 +236,10 @@ proc toMessageDto*(jsonObj: JsonNode): MessageDto =
   discard jsonObj.getProp("albumImagesCount", result.albumImagesCount)
   discard jsonObj.getProp("editedAt", result.editedAt)
   discard jsonObj.getProp("deleted", result.deleted)
+  discard jsonObj.getProp("deletedBy", result.deletedBy)
+  if result.deleted and result.deletedBy == "":
+    # The message was deleted by the sender itself
+    result.deletedBy = result.`from`
```

Comment:
Yes, because that code in `onMessageDeleted` is called from a status-go signal that only contains the messageID, the chatID and possibly the deletedBy (see `RemovedMessageDto`).